src/core/geometry/qgsrectangle.h

```diff
@@ -609,10 +608,10 @@ class CORE_EXPORT QgsRectangle
 
   private:
 
-    double mXmin = 0.0;
-    double mYmin = 0.0;
-    double mXmax = 0.0;
-    double mYmax = 0.0;
+    double mXmin = std::numeric_limits<double>::max();
```

Comment:
Should we not use  std::numeric_limits<double>::quiet_nan() instead of std::numeric_limits<double>::max()?

It seems more logical to me.